src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -265,7 +265,13 @@ const ReleaseContent = () => {
                   {release.coverageTitle} <strong>{release.yearTitle}</strong>
                 </p>
                 <Details summary={`See other releases (${releaseCount})`}>
-                  <ul className="govuk-list">
+                  <ul
+                    className="govuk-list"
+                    style={{
+                      maxHeight: '300px',
+                      overflowY: 'scroll',
```

Comment:
`auto` would be better than `scroll` for the overflow value as then the scrollbar will only show when the list is long enough to require scrolling.